Claudia/src/main/java/com/endava/task/second/Order.java

```diff
@@ -0,0 +1,39 @@
+package com.endava.task.second;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Random;
+
+public class Order {
+    public long id;
+    public LocalDateTime dateTime;
+    public User user;
+    public List<Item> items;
+
+    public Order(User user, List<Item> items){
+        this.id = Math.abs(new Random().nextLong());
+        this.dateTime = LocalDateTime.now();
+        this.user = user;
+        this.items = items;
+
+    }
+
+    public boolean containsClothes() {
+        for (Item item : this.items) {
+            if (item.getType() == Type.CLOTHES) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
I sugest to be more generic here by giving a Type parameter to method and check if item.getType() == type. This will ensure more flexibility to you code

Reply:
Done.